pom.xml

```
@@ -28,7 +28,7 @@
 
         <structured-logging.version>1.9.4</structured-logging.version>
         <sdk-manager-java.version>1.5.1</sdk-manager-java.version>
-        <private-api-sdk-java.version>2.0.77</private-api-sdk-java.version>
+        <private-api-sdk-java.version>unversioned</private-api-sdk-java.version>
```

```suggestion
        <private-api-sdk-java.version>2.0.78</private-api-sdk-java.version>
```
The version number increments. We can assume the new version will be 2.0.78 once https://github.com/companieshouse/private-api-sdk-java/pull/107 has been merged. Unless another merge happens before hand.

---

Good suggestion, but the SDK is updated by other teams fairly often. We could use the next increment by default and would only need to a further PR if necessary.
Using `unversioned` does allow devs to check out the branch for review purposes though. If the POM has a 'future' version, they would need to change it themselves.